documentation/create_overview_doc/createdoc.sh

```diff
@@ -554,11 +321,14 @@ function main() {
     module_files=$(eval $(create_find_command "$lang_csv" "$folder_to_document"))
 
     # Check if module_files is empty and display a warning if no files are found
-    [ -z "$module_files" ] && echo "Warning: No files found for documentation generation." && return
+    if [ -z "$module_files" ]; then
+        echo "Warning: No files found for documentation generation." >&2
+        return
+    fi
```

Comment:
**Suggestion**: Enhance user feedback by specifying which file types or extensions were targeted for documentation generation, especially when no files are found, to aid in troubleshooting. <br> **Code Suggestion**: 
 ```
 + echo "Warning: No files found for documentation generation. Targeted file extensions include: [list extensions here]." >&2
 ```